tests/conftest.py

```diff
@@ -1272,8 +1272,16 @@ def hosts_common_available_ports(nodes_available_nics):
 
     will return ['ens3', 'ens6']
     """
-    nics_list = list(set.intersection(*[set(_list) for _list in nodes_available_nics.values()]))
-    nics_list.sort()
+    nic_sets = [set(lst) for lst in nodes_available_nics.values()]
+    if not nic_sets:
+        LOGGER.warning("No common NICs across workers: nodes_available_nics is empty.")
```

Comment:
The warning message is misleading. If `nic_sets` is empty, it doesn't mean there are no **common NICs across the workers**. It means there are no available NIC in any if the workers.
Common NICs is what this fixture is trying to find.
But I won't require you to change it, do it only if you push a new commit. It's not worth having another round of reviews just for that.